modules/cms/classes/Theme.php

```diff
@@ -148,8 +148,12 @@ public function isActiveTheme()
     public static function getActiveThemeCode()
     {
         $activeTheme = Config::get('cms.activeTheme');
+        $themes = static::all();
```

Comment:
@vojtasvoboda I believe the `$themeHasChanged` check is redundant. If we have multiple available themes, we have no choice but to confirm the active theme with the database.

Could you change this line to the following and remove line 152-154:
```suggestion
        $hasMultipleThemes = count(static::all()) > 1;
```

Reply:
@vojtasvoboda Please ignore this for now. I'm going to make a different suggestion in the PR.